docs/server.rst

```diff
@@ -147,7 +158,7 @@ If we assume that you are installing Tunneldigger under ``/srv/tunneldigger``
     virtualenv -p /usr/bin/python3 env_tunneldigger
 
 .. note::
-    Tunneldigger only supports Python 3.
+    Latest Tunneldigger only supports Python 3.
```

Comment:
It's confusing, because the "stable" version shown as v0.3.0 is working under python 2.7.

Maybe someone can delete the "stable" read the docs version or update this? how can something be stable if it runs under python 2.7?

Reply:
I think this is related to https://github.com/wlanslovenija/tunneldigger/issues/145?

Reply:
I don't think docs should retrace the history of all things that used to be supported in the past... *everything* the docs say only applies to latest tunneldigger. It would be more confusing to emphasize this here as that might be read to imply that some of the rest of the docs also apply to older tunneldigger, which they do not.